streams/src/test/java/org/apache/kafka/streams/tests/StreamsUpgradeTest.java

```diff
@@ -228,14 +252,16 @@ public GroupAssignment assign(final Cluster metadata, final GroupSubscription gr
                     final Subscription subscription = entry.getValue();
 
                     final SubscriptionInfo info = SubscriptionInfo.decode(subscription.userData()
-                        .putInt(0, LATEST_SUPPORTED_VERSION)
-                        .putInt(4, LATEST_SUPPORTED_VERSION));
+                                                                                      .putInt(0, LATEST_SUPPORTED_VERSION)
+                                                                                      .putInt(4, LATEST_SUPPORTED_VERSION));
 
                     downgradedSubscriptions.put(
                         entry.getKey(),
                         new Subscription(
                             subscription.topics(),
                             new SubscriptionInfo(
+                                LATEST_SUPPORTED_VERSION,
```

Comment:
why add `LATEST_SUPPORTED_VERSION` version here?

Reply:
Just because I deleted a constructor whose only purpose was to fill in this value. It seems trivial to add it at the call site, and then you don't have to have 15 constructors for the class (hyperbole, of course).

Reply:
Ah yeah that makes sense, nice cleanup as well.